### PR TITLE
BUGZ-354: Update Android and Windows build documentation to reflect latest changes

### PR DESCRIPTION
**Pull request by annabrewer**
https://highfidelity.atlassian.net/browse/BUGZ-354

Updated Windows build instructions to reflect the fact that the C++ tools for Visual Studio are still needed, and added support for VS 2019.

Also updated the Android instructions to ask users to download the dependencies for their OS (e.g. C++ tools for VS on Windows). Made a few other changes: added instructions to create a keystore and run configuration for the app, updated the API levels, added the CMake version #, removed some outdated Gradle stuff, and wrote up a section on troubleshooting.

Lastly, merged BUILD_QUEST.md and BUILD_ANDROID.md, since they were nearly identical.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by ZappoMan** [comment]
add to whitelist

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19663/HighFidelity-Beta-PR15636-5c0d7c8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19663/HighFidelity-Beta-Interface-PR15636-5c0d7c8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19663/HighFidelity-Beta-PR15636-5c0d7c8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19663/HighFidelity-Beta-Interface-PR15636-5c0d7c8.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19663/HighFidelity-Beta-PR15636-5c0d7c8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19663/HighFidelity-Quest-Beta-PR15636-5c0d7c8.apk)

File: BUILD_ANDROID.md [hunk 1]
```diff
@@ -1,64 +1,118 @@
...
+
+#### For the Oculus Quest interface
+
+* From the _Launch_ drop down menu, select _Specified Activity_
+* In the _Activity_ field directly below, put `io.highfidelity.questInterface.PermissionsChecker`
+
+Note the 's' in Permission**s**Checker for the Quest.
+
+Now you are able to run your module! Click the green play button in the top toolbar of Android Studio
+r
```
Comment (jennaingersoll) [inline BUILD_ANDROID.md]: Remove this extra 'r'

File: BUILD_ANDROID.md [hunk 2]
```diff
@@ -1,64 +1,118 @@
...
+# Troubleshooting
+
+To view a more complete debug log,
```
Comment (jennaingersoll) [inline BUILD_ANDROID.md]: To view a more complete debug log: 

(note colon instead of comma)

File: BUILD_ANDROID.md [hunk 3]
```diff
@@ -1,64 +1,118 @@
...
+* Click the icon with the two overlapping squares in the upper left corner of the tab where the sync is running (hover text says _Toggle view_)
+* To change verbosity, click _File > Settings_. Under _Build, Execution, Deployment > Compiler_ you can add command-line flags, as per Gradle documentation
+
+Some things you can try if you want to do a clean build
+ 
+* Delete the `build` and `.externalNativeBuild` folders from the folder for each module you're building (for example, `hifi/android/apps/interface`)
+* If you have set your `HIFI_VCPKG_ROOT` environment variable, delete the contents of that directory; otherwise, delete `AppData/Local/Temp/hifi`
+* In Android Studio, click _File > Invalidate Caches / Restart_ and select _Invalidate and Restart_
+
+If you see lots of "couldn't acquire lock" errors,
```
Comment (jennaingersoll) [inline BUILD_ANDROID.md]: Again, use : instead of ,

File: BUILD_WIN.md [hunk 4]
```diff
@@ -1,18 +1,29 @@
...
 
-If you don’t have Community or Professional edition of Visual Studio 2017, download [Visual Studio Community 2017](https://www.visualstudio.com/downloads/).
+If you don’t have Community or Professional edition of Visual Studio, download [Visual Studio Community 2019](https://visualstudio.microsoft.com/vs/). If you have Visual Studio 2017, you are not required to download Visual Studio 2019.
 
-Leave default components.  
-If you do not already have a python development environment installed, also check  "Python Development" in this screen.
+When selecting components, check "Desktop development with C++". On the right on the Summary toolbar, select the following components.
 
-If you already have Visual Studio installed and need to add python, open the "Add or remove programs" control panel and find the "Microsoft Visual Studio Installer".  Select it and click "Modify".  In the installer, select "Modify" again, then check "Python Development" and allow the installer to apply the changes.
+#### If you're installing Visual Studio 2017,
```
Comment (jennaingersoll) [inline BUILD_WIN.md]: Remove comma

File: BUILD_WIN.md [hunk 5]
```diff
@@ -1,18 +1,29 @@
...
+
+* Windows 8.1 SDK and UCRT SDK
+* VC++ 2015.3 v14.00 (v140) toolset for desktop
+
+#### If you're installing Visual Studio 2019,
```
Comment (jennaingersoll) [inline BUILD_WIN.md]: Remove comma

File: BUILD_WIN.md [hunk 6]
```diff
@@ -29,7 +40,12 @@ Run Command Prompt from Start and run the following commands:
 `cd "%HIFI_DIR%"`  
 `mkdir build`  
 `cd build`  
-`cmake .. -G "Visual Studio 15 Win64"`  
+
+#### If you're using Visual Studio 2017,
```
Comment (jennaingersoll) [inline BUILD_WIN.md]: Remove comma

File: BUILD_WIN.md [hunk 7]
```diff
@@ -29,7 +40,12 @@ Run Command Prompt from Start and run the following commands:
...
+Run `cmake .. -G "Visual Studio 15 Win64"`.
+
+#### If you're using Visual Studio 2019,
```
Comment (jennaingersoll) [inline BUILD_WIN.md]: Remove comma

**Pull request merged**